jobs/update-stale-payment/requirements/prod.txt

```diff
@@ -15,5 +15,6 @@ jsonschema
 requests
 
 -e git://github.com/pwei1018/jaeger-client-python.git@186f14e14758273ed108508c0d388a4f4de5c75b#egg=jaeger-client
--e git+https://github.com/pwei1018/sbc-common-components.git#egg=sbc-common-components-1.0.0&subdirectory=python
-git+https://github.com/bcgov/sbc-pay.git#egg=pay-api&subdirectory=pay-api
+-e git+https://github.com/bcgov/sbc-common-components.git#egg=sbc-common-components-1.0.0&subdirectory=python
+# -e git+https://github.com/bcgov/sbc-pay.git#egg=pay-api&subdirectory=pay-api
+-e git+https://github.com/pksumesh/sbc-pay.git@delete_payment#egg=pay-api&subdirectory=pay-api
```

Comment:
the url looks like referring to a fork? is that okay?

Reply:
That will be changed in next PR, made that change to deploy in DEV as development won't have the latest changes to use in the job.